tests/core/conversion/converters/test_element_wise.cpp

```diff
@@ -261,13 +261,31 @@ TEST(Converters, ATenClampMaxConvertsCorrectly) {
 TEST(Converters, ATenClampMinMaxConvertsCorrectly) {
   const auto graph = R"IR(
   graph(%x.1 : Tensor):
-          %2 : int = prim::Constant[value=3]()
-          %3 : int = prim::Constant[value=-2]()
```

Comment:
Do we want a new negative number case?

Reply:
I don't think so, let the input range larger([1,5])than min(1.5)/max(3.5) will cover all situation

Reply:
ok